infrastructure/helm/keycloak/keycloak-setup.md

```diff
@@ -0,0 +1,64 @@
+# Install Quantumserverless and Configure Keycloak
+## In the command line
+Switch default namespace to the install target namespace
+```
+kubectl config set-context –current –namespace="target namespace"
+```
+Install helm chart
+```
+helm install "release name" quantemserverless
+```
+Wati for all pods come up except `releasename-kuberay-head-xxxx`
+Run ./login-setup.sh <RELEASE NAME>
```

Comment:
We think that maybe this script can be removed in favour of the next configuration:

- User and Password can be configured through environment variables as it says in the documentation, [here](https://www.keycloak.org/server/configuration#_setup_of_the_initial_admin_user).

- KEYCLOAKPORT & LOCALIP: is only needed to give to the user the URL to log-in?

Reply:
The initial admin user id and password can be set by adding 
```
keycloak:
  auth:
    adminUser: admin
    adminPassword: passw0rd
```
in the value.yaml file.  Thanks!